File: Feitos em C/2685.c
#include <stdio.h>
int main() {
	int x;
	while(scanf("%d",&x)==1){
		if((x>=0&&x<90)||x==360){
			printf("Bom Dia!!\n");
		}
		else if(x>=90&&x<180){
			printf("Boa Tarde!!\n");
		}
		else if(x>=180&&x<270){
			printf("Boa Noite!!\n");
		}
		else{
			printf("De Madrugada!!\n");
		}
	}
    return 0;
}

File: Feitos em C Sharp/2685.cs
using System;
namespace ConsoleApp18 {
    class Program {
        static void Main(string[] args) {
			while (true) {
				string x1 = Console.ReadLine();
				if (string.IsNullOrEmpty(x1)) {
					break;
				}
				int x = int.Parse(x1);
				if ((x >= 0 && x < 90) || x == 360) {
					Console.WriteLine("Bom Dia!!");
				}
				else if (x >= 90 && x < 180) {
					Console.WriteLine("Boa Tarde!!");
				}
				else if (x >= 180 && x < 270) {
					Console.WriteLine("Boa Noite!!");
				}
				else {
					Console.WriteLine("De Madrugada!!");
				}
			}
		}
    }
}